### PR TITLE
feat: new onboarding screens

### PR DESCRIPTION
**Pull request by mortennordseth**
FRAM would like to have some additional information in the app's onboarding to communicate the differences between the current app and the new one. I decided to use remote config to enable this. 

Before merging we need to release a new version of @atb-as/generate-assets and upgrade the dependency because of new illustrations (https://github.com/AtB-AS/design-system/pull/135). This will fix the tsc error.
<details>
<summary>Screenshots</summary>

image

image

</details>

Closes https://github.com/AtB-AS/kundevendt/issues/4066

### CODE REVIEW

File: src/translations/screens/Onboarding.ts [hunk 1]
```diff
@@ -21,6 +21,22 @@ const OnboardingTexts = {
     ),
     mainButton: _('Neste', 'Next'),
   },
+  goodToKnow: {
+    title: _('Greit å vite', 'Good to know'),
+    description: _(
+      'Når du gjør et reisesøk får du opp forslag til hvordan du kan reise, og en korrespondanse som er garantert er merket med korrespondanse.',
```
Comment (adriansberg) [inline src/translations/screens/Onboarding.ts:27]:
```suggestion
      'Når du gjør et reisesøk får du opp forslag til hvordan du kan reise, og en korrespondanse som garantert er merket med korrespondanse.',
```

**Comment by adriansberg** [comment]
Looks good. I'll run the app when the newly added assets ase added.

**Comment by adriansberg** [comment]
Different line thicknesses in the illustrations. Should they be the same or does it not matter?

image
vs
image

**Review by adriansberg — APPROVED** [review]
🚀

File: src/translations/screens/Onboarding.ts (hunk 1 above)
Comment (mortennordseth) [reply src/translations/screens/Onboarding.ts:27]: I think we should leave this as is or ask Christoffer for a new text if it is unclear. They want to state clearly that only guaranteed correspondences are marked with correspondence

Comment (adriansberg) [reply src/translations/screens/Onboarding.ts:27]: Okay. Think I understood the sentence now after reading it six times. Suggested reword by AI: "Når du søker etter reisemuligheter, vil du få forslag til ulike reisealternativer, og korrespondansen som er bekreftet vil være tydelig merket."

**Comment by mortennordseth** [comment]
> Different line thicknesses in the illustrations. Should they be the same or does it not matter?

That might be better, yes! I'll look into it 👍

**Comment by mortennordseth** [comment]
A new illustration will be included in https://github.com/AtB-AS/kundevendt/issues/4141.

**Pull request merged**